src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacSymbolicChemistryValidator.java

```diff
@@ -324,9 +324,7 @@ public QuestionValidationResponse validateQuestionResponse(final Question questi
             if (containsError) {
 
                 // User input contains error terms.
-                // FIXME: This currently clashes with determining whether the submitted answer was the wrong type
-                // Inequality should be changed to not allow Nuclear syntax in Chemistry questions and vice versa
-                feedback = new Content("Your answer contains invalid syntax!");
+                feedback = new Content("Your answer is not chemically valid!");

```

Comment:
I've always thought this warning was a bit "in your face" and certain. It's now even more certain, and yet I think that it's just as likely we don't support whatever they are trying to do. For numeric questions we use a softer tone: "Your answer is not in a format we recognise".

We could do something similar here: "Your answer is not in a format we recognise; it might not be chemically valid."?